sktime/transformations/series/summarize.py

```diff
@@ -323,27 +323,33 @@ def _transform(self, X, y=None):
             bfill = True
         else:
             bfill = False
+        concat_flag = False
```

Comment:
Sorry my first guess seems to be a mistake. Thus, I think the changes in this method should be reverted.